packages/white-whale-std/src/pool_manager.rs

```diff
@@ -253,10 +251,25 @@ pub enum QueryMsg {
     //     ask_amount: Uint128,
     //     operations: Vec<SwapOperation>,
     // },
-    #[returns(PairInfo)]
+    #[returns(PairInfoResponse)]
     Pair { pair_identifier: String },
 }
 
+#[cw_serde]
+pub struct ConfigResponse {
+    pub config: Config,
+}
+
+#[cw_serde]
+pub struct SwapRoutesResponse {
+    pub swap_routes: Vec<SwapRoute>,
+}
+
+#[cw_serde]
+pub struct PairInfoResponse {
+    pub pair_info: PairInfo,
+}
+
```

Comment:
What do we create these structs for? Won't they always only ever hold the single value?